wesnoth-low.po

```diff
@@ -3251,8 +3252,8 @@ msgid ""
 "Some weeks later, as Kalenz and his companions were working their way "
 "southwards from the high peaks of the Heart Mountains into its foothills..."
 msgstr ""
-"Po několika týdnech cesty na jih dorazil Kalenz se svými souputníky ze srdce "
-"velehor do pahorkatiny na úpatí..."
+"Po několika týdnech cesty na jih dorazil Kalenz se svými souputníky z "
+"vysokých vrcholů Srdcových hor do pahorkatiny na úpatí..."
```

Comment:
řekl bych, že u vrcholu, stejně jako u lorda, to high znamená další umocnění - vizte Elvish High Lord na [mé stránce](https://emer.cz/michal/wesnoth.html#sloh)
```suggestion
"nejvyšších vrcholů Srdcových hor do pahorkatiny na úpatí..."
```